pixels-index/pixels-index-rockset/src/test/java/io/pixelsdb/pixels/index/rockset/TestRocksetIndex.java

```diff
@@ -1,117 +1,117 @@
-/*
- * Copyright 2025 PixelsDB.
- *
```

Comment:
Why comment out the unit tests? Are there new unit tests for the new RocksetIndex implementation?